addons/l10n_mx/tests/test_account_move.py

```diff
@@ -0,0 +1,35 @@
+from odoo.tests import tagged
+
+from odoo import Command
+from odoo.addons.l10n_mx.tests.common import TestMxCommon
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestAccountMove(TestMxCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+    def test_credit_note_assign_right_account_on_lines(self):
+        """
+        This test check if the lines of a credit note created for MX, reference the
+        right account id by default.
+        """
+        partner_mx = self.env['res.partner'].create({
+            'name': 'Partner MX',
+            'country_id': self.env.ref('base.mx').id,
+        })
```

Comment:
put the helper from l10n_mx_edi into your common test suite to get the partner_mx